test/readme.md

```diff
@@ -357,6 +361,23 @@ The following steps will walk you through how to run the tests for CSFLE.
       }
    }
    ```
+1. Start the KMIP servers:
+
+  `DRIVERS_TOOLS="/path/to/mongodb-labs/drivers-evergreen-tools" .evergreen/run-kms-servers.sh`
+
+1. Ensure default ~/.aws/config is present:
+
+  ```
+  aws_access_key_id = AWS_ACCESS_KEY_ID
+  aws_secret_access_key = AWS_SECRET_ACCESS_KEY
+  ```
+
+1. Set temporary AWS credentials
+
+  ```
+  pip3 install boto3
+  PYTHON="python3" source /path/to/mongodb-labs/drivers-evergreen-tools/.evergreen/csfle/set-temp-creds.sh
+  ```
```

Comment:
Would you mind updating these docs to account for fish shell users too?  neal and I use fish.  I modified the script in drivers evergreen tools, we should just add a subsection saying "if you use fish, you can use the following script instead" or something

```fish
function set_aws_creds
    set PYTHON_SCRIPT "\
import boto3
client = boto3.client('sts')
credentials = client.get_session_token()['Credentials']
print (credentials['AccessKeyId'] + ' ' + credentials['SecretAccessKey'] + ' ' + credentials['SessionToken'])"

    echo $PYTHON_SCRIPT | python3 -
end

set CREDS (set_aws_creds)

set CSFLE_AWS_TEMP_ACCESS_KEY_ID (echo $CREDS | awk '{print $1}')
set CSFLE_AWS_TEMP_SECRET_ACCESS_KEY (echo $CREDS | awk '{print $2}')
set CSFLE_AWS_TEMP_SESSION_TOKEN (echo $CREDS | awk '{print $3}')

set -e CREDS
```

Reply:
Added.

Reply:
Thanks!